inspire/modules/formatter/views.py

```diff
@@ -47,3 +47,28 @@ def get_latex(recid, latex_format):
     record = get_record(recid)
     latex = Latex(record, latex_format).format()
     return jsonify({"result": latex, "recid": recid})
+
+
+@blueprint.route("/download-bibtex/<int:recid>")
+def get_bibtex_file(recid):
+    record = get_record(recid)
+    results = Bibtex(record).format()
+    generator = (cell for row in results
```

Comment:
Strange :/ in my machine is shows that is sent correctly! Nevermind, I am
 attaching you the bibtex file. If you open it with sublime you can see the
bibtex printed as we want.